src/gazebo_gps_plugin.cpp

```diff
@@ -38,6 +38,28 @@ GpsPlugin::~GpsPlugin()
       updateConnection_->~Connection();
 }
 
+bool GpsPlugin::checkWorldHomePosition(physics::WorldPtr world) {
+#if GAZEBO_MAJOR_VERSION >= 9
+  common::SphericalCoordinatesPtr spherical_coords = world_->SphericalCoords();
+#else
+  common::SphericalCoordinatesPtr spherical_coords = world_->GetSphericalCoordinates();
+#endif
+
+  if (!spherical_coords) {
+    return false;
+  }
+  world_latitude_ = spherical_coords->LatitudeReference().Degree() * M_PI / 180.0;
+  world_longitude_ = spherical_coords->LongitudeReference().Degree() * M_PI / 180.0;
+  world_altitude_ = spherical_coords->GetElevationReference();
+  //This logic is required since the spherical coordinates return 0 if the spherical coordnates
+  // are not defined in the world file
+  if (world_latitude_ && world_longitude_ && world_altitude_) {
```

Comment:
A one liner can make sense here:
```suggestion
  return (world_latitude_ && world_longitude_ && world_altitude_) ? true : false;
```